test/integration/targets/s3_bucket/tasks/main.yml

```diff
@@ -0,0 +1,255 @@
+---
+# tasks file for test_s3
+#   Test creating a bucket                                      (bucket1 created)
+#   Test creating a bucket with tags                            (bucket2 created)
+#   Test modifying tags of a bucket                             (bucket2 modified)
+#   Test creating a bucket with policy                          (bucket3 created)
+#   Test doing something not permitted by the policy            (bucket3 not modified)
+#   Test creating a bucket that already exists                  (bucket3 not modified)
+#   Test removing a policy from a bucket                        (bucket3 modified)
+#   Test deleting a bucket that doesn't exist                   (no bucket affected)
+#   Test deleting a bucket with an object in it without force   (bucket1 not modified)
+#   Test deleting a bucket with an object in it with force      (bucket1 deleted)
+#   Test deleting an empty bucket without force                 (bucket2 deleted)
+#   Test deleting an empty bucket with force                    (bucket3 deleted)
+#   Test creating a bucket with a . in the name                 (bucket4 created)
+#   Test deleting a bucket with a . in the name                 (bucket4 deleted)
+# ============================================================
+- name: generate random name for the bucket name
+  command: bash -c 'echo ansible_test_$RANDOM'
```

Comment:
Fix this

Reply:
@s-hertel I'm not clear from your comment whether this needs to happen before or after merge (If after, perhaps adding a FIXME comment would help)

Reply:
But really, using bash for this?

```
  set_fact:
    bucketname: "ansible_test_{{ 32767 | random }}"
```

Reply:
Also, you might want to fail just after this task if the bucket accidentally exists (or retry X times)

Reply:
@willthames It was just a note for myself. I need to use resource_prefix instead.

Reply:
Definitely before merging.